lib/scrabble.rb

```diff
@@ -1,4 +1,146 @@
 module Scrabble
-  # require your gems and classes here
-  # require_relative 'scrabble/whatevs'
+
+  # This is a Scrabble scoring system
+
+  SCORE_TABLE =   {
+    "A" => 1,
+    "E" => 1,
+    "I" => 1,
+    "O" => 1,
+    "U" => 1,
+    "L" => 1,
+    "N" => 1,
+    "R" => 1,
+    "D" => 2,
+    "G" => 2,
+    "B" => 3,
+    "C" => 3,
+    "M" => 3,
+    "P" => 3,
+    "F" => 4,
+    "H" => 4,
+    "V" => 4,
+    "W" => 4,
+    "Y" => 4,
+    "K" => 5,
+    "J" => 8,
+    "X" => 8,
+    "Q" => 10,
+    "Z" => 10,
+  }
+
+  class Scrabble
+
+#_______ Handling individual words _______
+
```

Comment:
I think that these methods would be a good opportunity to use `private` since you don't necessarily want to expose them all to the user of your scoring functionality.